src/unity/toolkits/object_detection/object_detector.cpp

```diff
@@ -320,7 +320,25 @@ variant_map_type object_detector::evaluate(
     std::map<std::string, flexible_type> opts) {
 
   // TODO: Support user-configurable metric and options.
-
+  std::vector<std::string> metrics;
+  static constexpr char AP[] = "average_precision";
+  static constexpr char MAP[] = "mean_average_precision";
+  static constexpr char AP50[] = "average_precision_50";
+  static constexpr char MAP50[] = "mean_average_precision_50";
+  std::vector<std::string> all_metrics = {AP,MAP,AP50,MAP50};
+  if (std::find(all_metrics.begin(), all_metrics.end(), metric) != all_metrics.end() ){
```

Comment:
Nitpick: While we don't have a strict style guide, I think that we should strive to make the formatting of our code somewhat consistent. The series of if statements here vary on a number of points:
* Whether there's a space between the `if` and the opening parenthesis
* Whether there's a space before the closing parenthesis of the `if`
* The indent level of the closing curly brace

FWIW, the other code in this file:
* Exactly one space immediately after `if` or `else`
* One space between a `)` and `{`
* Closing `}` on its own line (especially when the opening `{` is at the end of a line)
* Closing `}` at the same indent level as the `if` or `else` that opened that scope
* Generally, no space immediately after `(` or immediately before `)`. But definitely the opening and closing spacing should match one another

Reply:
(Someday maybe this can be something we do automatically as a post-processing step, using `clang-format`)